pkg/collector/corechecks/net/networkv2/network.go

```diff
@@ -662,45 +714,50 @@ func runCommand(cmd []string, env []string) (string, error) {
 }
 
 func collectConntrackMetrics(sender sender.Sender, conntrackPath string, useSudo bool, procfsPath string, blacklistConntrackMetrics []string, whitelistConntrackMetrics []string) {
-	if conntrackPath != "None" {
```

Comment:
the guard clause should only be fore the [ConntrackStats metrics](https://github.com/DataDog/integrations-core/blob/master/network/datadog_checks/network/check_linux.py#L273-L274) and not subsequent files (might be a bug, but this is a bug for bug migration of behavior)